packages/app-shell/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@norges-domstoler/app-shell",
+  "version": "1.0.0",
+  "description": "Wrapper for Lovisa applications.",
+  "author": "Elsa team",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/domstolene/designsystem"
+  },
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "homepage": "https://design.domstol.no",
+  "scripts": {
+    "build": "tsup --dts",
+    "build:fast": "tsup",
+    "dev": "pnpm build:fast -- --watch",
+    "typecheck": "tsc --noEmit"
+  },
+  "keywords": [
+    "dds",
+    "app-shell",
+    "design system",
+    "domstolene",
+    "norges domstoler",
+    "elsa"
+  ],
+  "devDependencies": {
```

Og som dev dependency:

```suggestion
  "devDependencies": {
    "@norges-domstoler/dds-components": "workspace:*",
```